client/src/components/App.jsx

```diff
@@ -14,11 +14,12 @@ import './styles/ActivityCard.css';
 export default function App() {
   const [places, setPlaces] = useState([]);
   const [activities, setActivities] = useState([]);
-  const [pageNumb, setPageNumb] = useState(1);
-  const [pageNumbActivs, setPageNumbActivs] = useState(1);
+  const childRef = useRef(null);
```

Comment:
this is impressive!!!! at least it is for me. i briefly read about this, but didn't know how to use it confidently. way to go!